spec/operators/combineLatest-spec.ts

```diff
@@ -1,480 +1,551 @@
 import { of } from 'rxjs';
 import { combineLatest, mergeMap, distinct, count } from 'rxjs/operators';
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
+import { TestScheduler } from 'rxjs/testing';
+import { observableMatcher } from '../helpers/observableMatcher';
 
 declare function asDiagram(arg: string): Function;
 
 /** @test {combineLatest} */
 describe('combineLatest operator', () => {
+  let testScheduler: TestScheduler;
+
+  beforeEach(() => {
+    testScheduler = new TestScheduler(observableMatcher);
+  });
+
   asDiagram('combineLatest')('should combine events from two cold observables', () => {
-    const e1 =   cold('-a--b-----c-d-e-|');
-    const e2 =   cold('--1--2-3-4---|   ');
-    const expected = '--A-BC-D-EF-G-H-|';
+    testScheduler.run(({ cold, expectObservable }) => {
+      const e1 = cold('  -a--b-----c-d-e-|');
+      const e2 = cold('  --1--2-3-4---|   ');
+      const expected = '--A-BC-D-EF-G-H-|';
```

Comment:
I think the original version was misaligned (you will find a bit of that in the code base). All of the snippets should start at the same time.